.github/ISSUE_TEMPLATE/story.md

```diff
@@ -38,13 +38,10 @@ As a _, so that _, I need _.
 - [ ] Code quality tasks completed
   - [ ] Code refactored for clarity and no design/technical debt
   - [ ] Adhere to separation of concerns; code is not tightly coupled, especially to 3rd party dependencies
-  - [ ] Code is reviewed or developed by pair; 1 approval is needed but consider requiring an outside-the-pair reviewer
-  - [ ] Code quality checks passed
-- [ ] Security & Privacy tasks completed
```

Comment:
Same with this one. Would deleting this entry lead to overlooking potential vulnerabilities or privacy concerns?

Reply:
@jorg3lopez or others, do you think there is a security or privacy gate or other security check that we aren't automatically checking during the PR process that we should consider as part of this DoD item?

Reply:
I like that we have the process as part of our PR review. For the most part, I think we are doing a great job using workflows and other software to test and detect vulnerabilities. Since we can't merge the PRs if they don't pass those security gates, I think it's ok to delete them here, as long as we keep them as part of the git checks.